CONTRIBUTING.md

```diff
@@ -12,13 +14,13 @@ If at any point you get stuck or want to ask questions, feel free to [open an is
 
 ---
 
-Contents:
+## Contents:
 1. [Creating an author profile](#creating-an-author-profile)
 2. [Writing a new post](#writing-a-new-post)
 3. [Translating a Blog Post](#translating-a-blog-post)
 4. [Submitting a Pull Request](#submitting-a-pull-request)
 
-# Creating an author profile
+# Creating an Author Profile
```

Comment:
We generally have a rule for headings (and we need to do better at enforcing this in blog posts) that headings should be lowercase outside of the first letter. This is something that much of the editing team here has agreed on and carries over from more professional editorial settings I've been a part of